wp-content/themes/oceanwp-child/social_share/share.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+global $post;
```

Comment:
Please wrap this part in a function with a meaningful name.
Also, the filename is not descriptive to (what I think) this is doing. Also change the file's name.

Reply:
Removed the file, I don't really need it.